.github/workflows/scripts/check-if-issue-and-pr-exist.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+PR_TITLE=$1
+ISSUE_TITLE=$2
```

Comment:
Document a usage example that shows how to use these inputs, and also document that `$GITHUB_TOKEN` is expected to be present in the environment.

Reply:
Oh, and document the need for `$REPO` to be present also.

Reply:
Done.